x/compute/client/cli/query.go

```diff
@@ -444,13 +444,30 @@ func GetCmdGetContractStateSmart(cdc *codec.Codec) *cobra.Command {
 			if err != nil {
 				return err
 			}
+			nonce := queryData[:32]
 
 			res, _, err := cliCtx.QueryWithData(route, queryData)
 			if err != nil {
+				if strings.Contains(err.Error(), "wasm contract failed: generic: ") {
```

Comment:
Wouldn't it be better to wrap it in a function?

Reply:
It would... 🤣